apps/dashboard/src/lib/utils/types/sidebar.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { ComponentProps } from 'svelte';
+import * as Sidebar from '@cio/ui/base/sidebar';
+import type { CurrentOrg } from './org';
+
+export interface SidebarData {
+  teams: CurrentOrg[];
+  navMain: Array<{
+    title: string;
+    url: string;
+    icon: any;
+    isActive: boolean;
+    items?: {
+      title: string;
+      url: string;
+    }[];
+  }>;
+  user: {
+    name: string;
+    email: string;
+    avatar: string;
+  };
+}
+
+export interface AppSidebar extends ComponentProps<typeof Sidebar.Root> {
```

Comment:
Is this type used in multiple places? if not then it shouldn't be in the global utils folder.

Instead keep it with the component using it by creating a types file as a sibling to the component.